fiftyone/brain/__init__.py

```diff
@@ -248,6 +250,11 @@ def compute_uniqueness(
         progress (None): whether to render a progress bar (True/False), use the
             default value ``fiftyone.config.show_progress_bars`` (None), or a
             progress callback function to invoke instead
+        similarity_backend (None): the similarity backend to use. The supported values are
+            ``fiftyone.brain.brain_config.similarity_backends.keys()`` and the
+            default is ``fiftyone.brain.brain_config.default_similarity_backend``
+        similarity_index (None): the similarity index to use. This can be computed using
```

Comment:
It would be nice to allow the user to provide either a `SimilarityIndex` object in-memory OR the `brain_key` string that can be used to load the index via:

```py
similarity_index = samples.load_brain_results(similarity_index)
```

Reply:
Allowing the user to provide the string `brain_key` makes this method easier to use in [operators](https://docs.voxel51.com/plugins/developing_plugins.html#developing-operators), which I can explain offline sometime.

Reply:
Added support for setting `similarity_index` to either `SimilarityIndex` or brain_key string